english/beach/beach.md

```diff
@@ -0,0 +1,5 @@
+You know in your heart the beach is where you wanna be. The sun, the sand, the sea and the breeze. You close your eyes and you can just imagine the beauty. Without any hesitation you head for the door.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.